packages/@aws-cdk/aws-apigatewayv2/test/http/api.test.ts

```diff
@@ -7,11 +7,14 @@ import { HttpApi, HttpMethod, LambdaProxyIntegration } from '../../lib';
 describe('HttpApi', () => {
   test('default', () => {
     const stack = new Stack();
-    const api = new HttpApi(stack, 'api');
+    const api = new HttpApi(stack, 'api', {
+      description: 'My Api',
+    });
 
     expect(stack).toHaveResource('AWS::ApiGatewayV2::Api', {
       Name: 'api',
       ProtocolType: 'HTTP',
+      Description: 'My Api',
```

Comment:
Could you add a separate test case for this? This doesn't match the 'default' test case.